ever_given/ever_given/wrapper.py

```diff
@@ -1,7 +1,113 @@
+import copy
+from pathlib import Path
+import shlex
+
 import docker
 
 
-def run_submission_container(container_uri, command):
+GUEST_INPUT_DIR = Path("/mnt") / "inputs"
+GUEST_OUTPUT_DIR = Path("/mnt") / "outputs"
+
+
+def _guest_input_path(filename):
+    return GUEST_INPUT_DIR / Path(filename).name
+
+
+def _prepare_commandline(command, args_dict):
+    return command + " " + " ".join(
```

Comment:
Haven't tested this but do you think it is more clear?
```suggestion
    return f"{command}  ".join(
```
Or am I misunderstanding what is going on here, command, 2 spaces, then `join`?